client/scss/components/_promo.scss

```diff
@@ -13,13 +11,55 @@
   @include container-query(('l': ('12'), 'm': ('8'))) {
     border-bottom: 0;
   }
+
+  .icon {
+    vertical-align: middle;
+    margin-right: map-get($gutter-width, 'small')/2;
+
+    @include respond-to('medium') {
+      margin-right: map-get($gutter-width, 'medium')/2;
+    }
+
+    @include respond-to('large') {
+      margin-right: map-get($gutter-width, 'large')/2;
+    }
+  }
 }
 
-.promo__image {
+.promo--gallery {
+  .icon { // aspect ratio: 24 x 19
+    width: (24/19) * 1.6em;
+    height: (19/19) * 1.6em;
+  }
+}
+
+.promo--podcast {
+  .icon { // aspect ratio: 27 x 22
+    width: (27/22) * 1.6em;
+    height: (22/22) * 1.6em;
+  }
+}
+
+.promo--video {
+  .icon { // aspect ratio: 17 x 19
+    width: (17/19) * 1.6em;
+    height: (19/19) * 1.6em;
+  }
+}
+
+.promo--underlined {
```

Comment:
Only prob I had with this was that the underline wasn't on the promo when it was in a `.grid__cell--l12`.